agent/consul/server_test.go

```diff
@@ -187,21 +185,20 @@ func testServerConfig(t *testing.T) (string, *Config) {
 	return dir, config
 }
 
+// Deprecated: use testServerWithConfig instead. It does the same thing and more.
 func testServer(t *testing.T) (string, *Server) {
-	return testServerWithConfig(t, func(c *Config) {
-		c.Datacenter = "dc1"
-		c.PrimaryDatacenter = "dc1"
-		c.Bootstrap = true
```

Comment:
These are already set in `testServerWithConfig`, so I don't think setting them here again does anything.